pkg/runtime/setup.go

```diff
@@ -486,7 +486,7 @@ func (s *setup) install(artifact *buildplan.Artifact) (rerr error) {
 		}
 		s.depot.Track(id, &deployment{
 			Type:  deploymentTypeEcosystem,
-			Path:  filepath.Join(s.path, artifact.ArtifactID.String()), // dummy path for uniqueness
+			Path:  s.path,
```

Comment:
I found out the depot actually validates deployments by checking if files exist in them. Any empty or non-existant deployments will be ignored.

So in order to load the injected `CLASSPATH` env var, we need valid deployments, including the `Files` field.